src/tests/Rocket.test.js

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import Rocket from '../components/Rocket';
+import store from '../redux/store';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('Test suite for the Rocket component', () => {
+  it('Should render a Snapshot of the Rocket component', () => {
+    const rocket = render(
+      <Provider store={store}>
+        <Rocket />
+      </Provider>,
+    );
+    expect(rocket).toMatchSnapshot();
+  });
+
+  test('Should test if the text "Reserve Rocket" is inside the Rocket component', () => {
+    render(
+      <Provider store={store}>
+        <Rocket />
+      </Provider>,
```

Comment:
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/65068771/224189498-35a97367-15d8-4dfe-9ac2-d7c832e88d16.png">


- [ ] Nice job writing tests using React Testing Library and jest, however, you need to work on solving the waring in your tests even though all your tests are passing. these warnings are appearing because you did not mock the required props for the `<Rocket/>` component. hint for removing them 👇 

```.test.js
render(
      <Provider store={store}>
        <Rocket id="id" name="name" description="des" img="img" reserved={false} />
      </Provider>,
    );
```